app/views/admin/lead_provider_users/edit.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-l">Edit user details</h1>
+    <%= form_for(@lead_provider_user, url: admin_lead_provider_user_path(@lead_provider, @lead_provider_user)) do |f| %>
+      <%= f.govuk_error_summary %>
+      <%= f.govuk_text_field(
+            :full_name,
+            value: @lead_provider_user.full_name,
+            label: { text: "Full name" })
+      %>
+      <%= f.govuk_email_field(
+            :email,
+            value: @lead_provider_user.email,
+            label: { text: "Email" })
+      %>
```

Comment:
That feels a bit boilerplatey. Aren't the fields clever enough to know what value the object has? 

Maybe we want `form_with model: @lead_provider_user`?

Reply:
That was my oversight. I'll correct it